src/FieldWarning/Assets/Model/Game/MatchSession.cs

```diff
@@ -131,6 +147,18 @@ public void Awake()
 #endif
         }
 
+        public void Start()
+        {
+            
+            if (_loadedData == null)
+            {
+                LoadingScreen.destinationScene = SceneManager.GetActiveScene().buildIndex;
+                SceneManager.LoadScene(4, LoadSceneMode.Single);
+            }
```

Comment:
else 
            {
#if UNITY_EDITOR
                // Default to hosting if entering play mode directly into a match scene:
                if (!NetworkClient.isConnected)
                    _networkManager.StartHost();
#endif
            }


+ remove the similar fragment from Awake()